infrastructure/server/resources/proxy.sub.pac

```diff
@@ -0,0 +1,7 @@
+function FindProxyForURL(url, host) {
+    if (dnsDomainIs(host, '.wpt.test')) {
+        return "PROXY 127.0.0.1:{{ports[http][0]}}"
```

Comment:
I wonder id we should expose the ip address in the config, so we can avoid hardcoding `127.0.0.1` here (not a blocking issue).

Reply:
Followup: https://github.com/web-platform-tests/wpt/pull/34689